packages/packlist/README.md

```
@@ -1,7 +1,6 @@
 # @outfitter/packlist
 
-> Unified development configuration manager that orchestrates all Outfitter
-> development tools
+> Unified development configuration manager that orchestrates all Outfitter development tools
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 80; Actual: 93

(MD013, line-length)